tests/model/test_spectrum.py

```diff
@@ -9,9 +9,11 @@
 
 from dxtbx.model import Spectrum
 
+from cctbx import factor_ev_angstrom
+
```

Comment:
should go just above the scitbx import, no extra newline.

(Can't make this a suggested code change because out of range)